cluster/images/obs-csi-plugin/socket-server.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"k8s.io/klog"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+const (
+	socketPath    = "/etc/csi-tool/connector.sock"
+	passwdFilePre = "passwd_file="
+	mountCmd      = "/mount"
+)
+
+type ResponseBody struct {
+	Data string
+}
+
+type MyHandler struct{}
+
+func (myHandler *MyHandler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
+	body, err := ioutil.ReadAll(request.Body)
+	if err != nil {
+		generateResponseBody(writer, http.StatusBadRequest, err.Error())
+	}
+	if len(body) == 0 {
+		generateResponseBody(writer, http.StatusBadRequest, "Request body is empty")
+	}
+	value := string(body)
+	path := request.URL.Path
+	klog.Infof("ServerHTTP Request body: %s， Request path: %s", value, path)
+
+	if path == mountCmd {
+		if err = mountHandler(value); err != nil {
+			generateResponseBody(writer, http.StatusInternalServerError, err.Error())
+		}
+		generateResponseBody(writer, http.StatusOK, "success")
+		return
+	}
+	generateResponseBody(writer, http.StatusInternalServerError, fmt.Sprintf("error request path: %s", path))
+}
+
+func generateResponseBody(writer http.ResponseWriter, statusCode int, data string) {
+	responseBody := ResponseBody{
+		Data: data,
+	}
+	response, _ := json.Marshal(responseBody)
+	writer.WriteHeader(statusCode)
+	writer.Header().Set("Content-Type", "application/json")
+	writer.Write(response) //nolint:errcheck
+}
+
+func main() {
+	flag.Parse()
```

Comment:
It looks like we're not using command line flags and no parsing is required.

Reply:
done